geniza/settings/components/base.py

```diff
@@ -291,3 +291,6 @@
 
 # default font base url
 FONT_URL_PREFIX = "/static/fonts/"
+
+# Taggit customization
+TAGGIT_TAGS_FROM_STRING = "geniza.common.utils.custom_tag_string"
```

Comment:
Please also turn on case-insensitive tag searching (I thought there was an issue for this but now I can't find it!)

```suggestion
TAGGIT_TAGS_FROM_STRING = "geniza.common.utils.custom_tag_string"
TAGGIT_CASE_INSENSITIVE = True 
```